modules/installation/pages/client-requirements.adoc

```diff
@@ -7,19 +7,23 @@
 Supported operating systems for traditional and Salt clients are listed in this table.
 
 [[mgr.supported.clients]]
-[cols="1,1,1,1", options="header"]
+[cols="1,1,1,1,1", options="header"]
 .Supported Client Systems
 |===
-| Operating Systems                                    | Architecture                                 | Traditional Clients | Salt Clients
-| {sle} 11 SP4                                         | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported           | Supported
-| {sle} 12 SP3, 12 SP4                                 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-| {sle} 15                                             | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-|_Latest minor release {rhnminrelease6}_               | {x86}, {x86}_64                              | Supported           | Supported
-|_Latest minor release {rhnminrelease7}_               | {x86}_64                                     | Supported           | Supported
-| Open Enterprise Server 2015, 2015 SP1, 2018          | {x86}_64                                     | Supported           | Supported
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 12 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {sle} 10 SP3+ | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {rhel} 8 | {x86}_64 | TBD | TBD
```

Comment:
As of now RHEL 8 is not supported. Either we remove this line for now, or we write something like future or comming soon.

Reply:
I've got a note   about what  TBD means now: ``Fields marked as `TBD` are under consideration, and may or may not be supported at a later date.``

Does this resolve the problem?

Reply:
Is this also clear for the Customer? For me `TBD` is a placeholder for developers and the decission should be done before it reach the customer.

Maybe we should write `under consideration` ?

But if this is a typical term you find also in other docs I am fine with it.

Reply:
TBD and TBC are both used fairly frequently in tech docs (in my experience, anyway), and we are explaining the term. I think  'Under consideration' is too long for the table. Alternatively, we could put a unicode question mark or something in, to match the style of the ticks and crosses?

Reply:
Ok, you are the experts. So I am fine with TBD or TBC.